config/vmdb.tmpl.yml

```diff
@@ -100,8 +100,6 @@ log:
       - log/apache/*.log
       - log/*.txt
       - config/*
-      - /opt/rh/postgresql92/root/var/lib/pgsql/data/*.conf
-      - /opt/rh/postgresql92/root/var/lib/pgsql/data/pg_log/*
```

Comment:
We can drop this from the yml and not migrate data because the log collection code was changed above to add these paths from ENV variables if the configuration didn't already have these paths.